src/app/zap-templates/zcl/data-model/chip/rvc-clean-mode-cluster.xml

```diff
@@ -30,7 +30,7 @@ limitations under the License.
   </enum>
 
   <cluster>
-    <domain>General</domain>
+    <domain>Robots</domain>
```

Comment:
What does this name refer to? Is it defended anywhere?

Reply:
@hicklin This shows up in ZAP, when it displays the groups of clusters.